python/lsst/ts/xml/data/sal_interfaces/MTMount/MTMount_Telemetry.xml

```diff
@@ -184,7 +184,7 @@
   </SALTelemetry>
   <SALTelemetry>
     <Subsystem>MTMount</Subsystem>
-    <EFDB_Topic>MTMount_cabinet0101</EFDB_Topic>
+    <EFDB_Topic>MTMount_cabinet0101Thermal</EFDB_Topic>
     <item>
       <EFDB_Name>actualTemperature</EFDB_Name>
       <Description>Actual surface temperature of cabinet 0101</Description>
```

Comment:
Please terminate the descriptions in a `.` here and in lines 5523 and 796.

Reply:
OK. The telemetry XML is auto-generated from the TMA's telemetry config file. I will update the generator.